sdk/python/kfp/cli/diagnose_me/kubernetes_cluster.py

```diff
@@ -38,6 +39,7 @@ class Commands(enum.Enum):
     Commands.GET_SECRETS: 'get secrets',
     Commands.GET_SERVICES: 'get services',
     Commands.GET_KUBECTL_VERSION: 'version',
+    Commands.GET_CONFIG_MAP: 'get configmap',
```

Comment:
Which configmap? KFP will have several of them.

Reply:
good point, changing the enum to GET_CONFIG_MAPS ... this will return all configs maps.